apps/web/components/booking/pages/BookingPage.tsx

```diff
@@ -51,12 +52,11 @@ type BookingFormValues = {
   };
 };
 
-const BookingPage = (props: BookingPageProps) => {
+const BookingPage = ({ eventType, booking, profile }: BookingPageProps) => {
```

Comment:
eventType was being deconstructed on line 58, booking and profile werent. I added all as props, so we don't have as many props.(...) in the file.